debug/linear-ease.html

```diff
@@ -0,0 +1,138 @@
+    <style>
+        body { margin: 0; padding: 0; }
+        html, body, #map { height: 100%; }
+
+        .mapboxgl-marker {
+            width: 10px;
+            height: 10px;
+            background: red;
+            margin-top: -5px;
+            margin-left: -5px;
+            border-radius: 5px;
+            cursor: pointer;
+        }
+
+        #centerpoint {
+            position: absolute;
+            top: 50%;
+            left: 50%;
+        }
+
+        #dot {
+            width: 30px;
+            height: 30px;
+            background-color: #444;
+            position: relative;
+            right: 15px;
+            bottom: 15px;
+            z-index: 1000;
+            border-radius: 100px;
+            border: 3px solid #FFF;
+            box-sizing: border-box;
+            box-shadow: 1px 1px 5px #444;
+        }
+    </style>
+</head>
+
+<body>
+<div id='map'></div>
+
+<script src='/dist/mapbox-gl-dev.js'></script>
+<script src='/debug/access-token-generated.js'></script>
+
+<script>
+var getCirclePoints = function(center, radiusInKm, points) {
+    if(!points) points = 64;
+
+    var km = radiusInKm;
+
+    var ret = [];
+    var distanceX = km/(111.320*Math.cos(center.lat*Math.PI/180));
+    var distanceY = km/110.574;
+
+    var theta, x, y;
+    for(var i=0; i<points; i++) {
+        theta = (i/points)*(2*Math.PI);
+        x = distanceX*Math.cos(theta);
+        y = distanceY*Math.sin(theta);
+
+        ret.push([center.lng+x, center.lat+y]);
+    }
+    ret.push(ret[0]);
+    return ret;
+};
+
+var map = new mapboxgl.Map({
+    container: 'map',
+    style: 'mapbox://styles/mapbox/streets-v9',
+    center: [-93.6, 41.6],
+    zoom: 5,
+    pitch: 60
+});
+
+map.on('load', function() {
+    var i=0;
+
+    var waypoints = getCirclePoints(map.getCenter(), 500, 16);
+
+    map.addSource("polygon", {
+        "type": "geojson",
+        "data": {
+            "type": "FeatureCollection",
+            "features": [{
+                "type": "Feature",
+                "geometry": {
+                    "type": "Polygon",
+                    "coordinates": [waypoints]
+                }
+            }]
+        }
+    });
+
+    map.addLayer({
+        "id": "polygon",
+        "type": "fill",
+        "source": "polygon",
+        "layout": {},
+        "paint": {
+            "fill-color": "blue",
+            "fill-opacity": 0.6
+        }
+    });
+
+    var duration = 3000;
+
+    var panToNextWaypoint = function() {
+        var waypoint = waypoints.shift();
+        waypoints.push(waypoint);
+
+        map.panTo(waypoint, {
+			duration: duration,
+			easing: function(t) {
+				return t;
+			}
+		});
+    };
+
+    setInterval(panToNextWaypoint, duration);
+
+	map.on('click', function() {
+  	if(map.getPitch() === 0) {
+    	map.setPitch(60);
+    } else {
+    	map.setPitch(0);
+    }
+  });
+});
+</script>
+<div id="centerpoint"><div id="dot"></div></div>
+</body>
+</html>
```

Comment:
It makes sense that you built a debug page to use while developing this feature. 

Because repository size, simplicity, and stability is important to us, we would prefer not to merge this file into `master`. Can you please remove this file from the PR?